modules/end-user-guide/examples/snip_che-configuring-a-workspace-to-use-an-intellij-based-ide.adoc

```diff
@@ -0,0 +1,55 @@
+.Procedure
+
+. When using a JetBrains IDE requesting a registration, see xref:provisioning-jetbrains-activation-code-for-offline-use.adoc[].
+
+. Add the following component to the workspace devfile:
++
+[source,yaml,subs="+quotes,macros,attributes"]
+----
+components:
+  - type: cheEditor
+    reference: "__<URL_to_meta.yaml>__" <1>
+----
++
+<1> __<URL_to_meta.yaml>__: HTTPS resource defining the editor metadata. It can be:
++
+* IntelliJ IDEA Community prebuilt image
++
+[source,yaml,subs="+quotes,macros,attributes"]
+----
+components:
+  - type: cheEditor
+    reference: https://raw.githubusercontent.com/che-incubator/jetbrains-editor-images/meta/che-idea/latest.meta.yaml
+----
++
+* PyCharm Community prebuilt image
++
+[source,yaml,subs="+quotes,macros,attributes"]
+----
+components:
+  - type: cheEditor
+    reference: https://raw.githubusercontent.com/che-incubator/jetbrains-editor-images/meta/che-pycharm/latest.meta.yaml
+----
++
+* Metadata for a custom editor image, see xref:building-intellij-editor-images.adoc[].
```

Comment:
* Custom-built IntelliJ editor image. See xref:building-intellij-editor-images.adoc[].

Reply:
There are three items in this list, and somehow they need to be made parallel/similar because currently the first two items are images and the third item is metadata, which is confusing.

Reply:
All three of  them are pointers to metadata.

Reply:
Fixed.